share/rpcauth/rpcauth.py

```diff
@@ -9,26 +9,36 @@
 import base64
 import hmac
 
-if len(sys.argv) < 2:
-    sys.stderr.write('Please include username as an argument.\n')
-    sys.exit(0)
+def generate_salt():
+    # This uses os.urandom() underneath
+    cryptogen = SystemRandom()
 
-username = sys.argv[1]
+    # Create 16 byte hex salt
+    salt_sequence = [cryptogen.randrange(256) for i in range(16)]
```

Comment:
nit: prefer `_` for unused loop variables.

Reply:
Done